### PR TITLE
🍿 [Portfolio] Wallet endpoint

### PR DESCRIPTION
**Pull request by marcinciarka**
- Added api/portfolio/wallet endpoint
- Added some necessary typing
- Debug logging on the new empty portfolio page

### CODE REVIEW

File: blockchain/networks/debank-network-names.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  [DebankNetworkNames.baseMainnet]: NetworkNames.baseMainnet,
+}
+
+export const OursNameToDebankNetworkName = {
+  [NetworkNames.ethereumMainnet]: DebankNetworkNames.ethereumMainnet,
+  [NetworkNames.arbitrumMainnet]: DebankNetworkNames.arbitrumMainnet,
+  [NetworkNames.polygonMainnet]: DebankNetworkNames.polygonMainnet,
+  [NetworkNames.optimismMainnet]: DebankNetworkNames.optimismMainnet,
+  [NetworkNames.baseMainnet]: DebankNetworkNames.baseMainnet,
+}
```
Comment (marcinciarka) [inline blockchain/networks/debank-network-names.ts:26]: Totally not sure about the naming here, also maybe it's possible to make it simpler?

File: features/portfolio/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const portfolioAPIUrl = new URL('https://pro-openapi.debank.com/')
```
Comment (piotrwitek) [inline features/portfolio/constants.ts:1]: should be env, we need to control per env

Comment (marcinciarka) [reply features/portfolio/constants.ts]: Fixed

File: pages/api/portfolio/wallet/[address].tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import type { DebankTokensReply, PortfolioAssetsReply } from 'features/portfolio/types'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { object, string } from 'zod'
+
+const portfolioWalletAddressSchema = object({
+  address: string(),
+})
+
+export default async function (req: NextApiRequest, res: NextApiResponse) {
+  switch (req.method) {
```
Comment (piotrwitek) [inline pages/api/portfolio/wallet/[address].tsx:13]: Just fail promptly when requested methoe not equal get, lesss indent

Comment (marcinciarka) [reply pages/api/portfolio/wallet/[address].tsx]: Done.

File: pages/api/portfolio/wallet/[address].tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    case 'GET':
+      const params = portfolioWalletAddressSchema.parse(req.query)
+      if (!process.env.DEBANK_API_URL) {
+        return res.status(500).json({
+          error: 'Missing DEBANK_API_URL env variable',
+        })
+      }
+      if (!params.address) {
+        return res.status(500).json({
```
Comment (piotrwitek) [inline pages/api/portfolio/wallet/[address].tsx]: I would put bad request here

**Review by piotrwitek — APPROVED** [review]

**Pull request merged**